packages/datadog-instrumentations/src/helpers/register.js

```diff
@@ -22,6 +22,8 @@ const disabledInstrumentations = new Set(
   DD_TRACE_DISABLED_INSTRUMENTATIONS ? DD_TRACE_DISABLED_INSTRUMENTATIONS.split(',') : []
 )
 
+const esmFirstLibraries = new Set(['vitest', '@vitest/runner'])
```

Comment:
I'm not sure about adding here libraries related data, should we move it to `hooks.js`?

```javascript
'@vitest/runner': { fn: () => require('../vitest'), esmFirst: true },
//...
vitest: { fn: () => require('../vitest'), esmFirst: true },
```

Reply:
I think that's a good idea

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/4415/commits/f006d71ee495fdb5ec63a5f8ab4ec24215addb25. I think it looks good @uurien